lom/src/pluginmgr/pluginmgr_main.go

```diff
@@ -52,7 +59,6 @@ import (
  */
 func main() {
     // setup logging
-    lomcommon.LogInit()
     lomcommon.SetLogLevel(syslog.LOG_INFO)
```

Comment:
Where passed in syslog level is parsed & set ?

Reply:
Drop -- Leave to Lib's default

Reply:
removed